src/settings/Settings.tsx

```diff
@@ -962,7 +962,7 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS,
         default: false,
     },
-    "videoChannelRoomId": {
+    "activeCallRoomId": {
```

Comment:
It's not worth the effort IMO, since this is in beta as you pointed out, and this flag only exists for the edge case where someone force-quits the app while on a call, which has other mitigations.

Reply:
Makes sense :+1: